dev/AppNotifications/AppNotificationBuilder/AppNotificationBuilder.cpp

```diff
@@ -281,30 +314,34 @@ namespace winrt::Microsoft::Windows::AppNotifications::Builder::implementation
         return wil::str_printf<std::wstring>(L"%ls%ls%ls", m_inlineImage.c_str(), m_heroImage.c_str(), m_appLogoOverride.c_str());
     }
 
-    std::wstring AppNotificationBuilder::GetButtons()
+    std::wstring AppNotificationBuilder::GetActions()
     {
-        if (m_buttonList.size())
-        {
-            std::wstring result{};
-            for (auto input : m_buttonList)
-            {
-                if (input.ButtonStyle() != AppNotificationButtonStyle::Default)
-                {
-                    m_useButtonStyle = true;
-                }
 
-                result.append(input.as<winrt::Windows::Foundation::IStringable>().ToString().c_str());
-            }
+        std::wstring result{};
+        for (auto input : m_textBoxList)
+        {
+            result.append(input.c_str());
+        }
 
-            return wil::str_printf<std::wstring>(L"<actions>%ls</actions>", result.c_str());
+        for (auto input : m_comboBoxList)
+        {
+            result.append(input.as<winrt::Windows::Foundation::IStringable>().ToString().c_str());
         }
-        else
+
+        for (auto input : m_buttonList)
         {
-            return {};
+            if (input.ButtonStyle() != AppNotificationButtonStyle::Default)
+            {
+                m_useButtonStyle = true;
+            }
+
+            result.append(input.as<winrt::Windows::Foundation::IStringable>().ToString().c_str());
         }
+
+        return (result.empty()) ? result : wil::str_printf<std::wstring>(L"<actions>%ls</actions>", result.c_str());
```

Comment:
NIT: `return result.empty() ? ...`

Reply:
Fixed in https://github.com/microsoft/WindowsAppSDK/pull/2813